db/sqlinterfaces.c

```diff
@@ -3352,7 +3352,7 @@ static int get_prepared_stmt_int(struct sqlthdstate *thd,
         clnt->prep_rc = rc = sqlite3_prepare_v3(thd->sqldb, rec->sql, -1,
                                                 sqlPrepFlags, &rec->stmt, &tail);
 
-        if (rc == SQLITE_OK) {
+        if (rc == SQLITE_OK && rec->stmt != NULL) {
```

Comment:
This is wrong, rc should not be OK and not have a sqlite vdbe ready

Reply:
I don't think I understood this. rec->stmt could still be NULL even when sqlite_prepare() returns SQLITE_OK (for empty statements for instance).